app/controllers/quotes_controller.rb

```diff
@@ -45,6 +45,11 @@ def destroy
     redirect_to quotes_path
   end
 
+  def best
+    @quotes = Quote.all.sort_by(&:score).reverse
+    @user_votes = current_user.votes
```

Comment:
You don't need to fix it here, but as you continually add @user_votes to most methods in here, you should start recognizing that it would be better to pull it out and handle it in a way that requires less duplication.